aisynphys/connectivity.py

```diff
@@ -237,6 +242,18 @@ def pair_was_probed(pair, synapse_type):
     qc_field = 'n_%s_test_spikes' % synapse_type
     return getattr(pair, qc_field) > 10
 
+def pair_probed_gj(pair):
+    """Return boolean indicateing whether a cell pair was "probed" for a gap junction.
+
+    Checks both the "pre" and "post" synaptic cells for long-pulse stimuli from which
+    gap junctions were identified and quantified.
+    """
+    
+    pre_electrode = pair.pre_cell.electrode
+    post_electrode = pair.post_cell.electrode
+    pre_stims = set([rec.stim_name for rec in pre_electrode.recordings])
+    post_stims = set([rec.stim_name for rec in post_electrode.recordings])
+    return any('TargetV' in s for s in pre_stims) and any('TargetV' in s for s in post_stims)
```

Comment:
Why check the postsynaptic cell?

Reply:
Given that we expect a gap junction to be reciprocal I wanted to check both. I was also getting more probes with just the pre-cell check than before with the synaptic check which didn't seem right